daemon/export_test.go

```diff
@@ -203,6 +204,19 @@ func MockSnapstateInstallPathMany(f func(context.Context, *state.State, []*snap.
 	}
 }
 
+func MockRebootAndCaptureArgs(rebootAction *boot.RebootAction,
```

Comment:
this is workable but a bit awkward, we tend not to do this,  we tend to take a function so that we can use closure in the tests and decide how to check/store the arguments